### PR TITLE
Pull request for project review

### PR DESCRIPTION
**Pull request by visva-dev**
## Design teardown
- Grayscale heatmap of SmashingMagazine website

### CODE REVIEW

**Comment by visva-dev** [comment]
## Design teardown
- Grayscale heatmap of SmashingMagazine website

**Comment by addod19** [comment]
> _No description provided._

Kindly write the description here instead. Thanks

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

File: index.html [hunk 1]
```diff
@@ -0,0 +1,433 @@
...
+        <li class="menu"><a href="#">Events</a></li>
+        <li class="menu"><a href="#">Membership</a></li>
+        <li class="menu"><a href="#">More</a></li>
+      </ul>
+      <form>
+        <input type="text" placeholder="Search articles..." />
+      </form>
+    </nav>
+    <div class="latest"><span>Don't Miss These Latest Articles</span></div>
+    <section class="top-content">
```
Comment (addod19) [inline index.html:37]: We could improve semantics by using ```<main>``` instead of ```<div class="top-content">```

**Comment by visva-dev** [comment]
## Changes made
- Fixed horizontal scroll issue
- Fixed article row section
- Refactored styles file
- Made HTML code more semantic

**Pull request merged**